src/Workspaces/CSharp/Portable/Extensions/SyntaxTokenExtensions.cs

```diff
@@ -34,23 +34,6 @@ public static bool IsLiteral(this SyntaxToken token)
             }
         }
 
-        public static bool IntersectsWith(this SyntaxToken token, int position)
-        {
-            return token.Span.IntersectsWith(position);
-        }
```

Comment:
note: i unified this with VB and moved to common layer.  there is nothing language specific with this particular extension.